static/js/click_handlers.js

```diff
@@ -635,6 +635,22 @@ exports.initialize = function () {
     });
 
     // COMPOSE
+    $("#compose_giphy_logo").on("click", () => {
+        $("#compose_box_giphy_grid").popover("toggle");
+        let gifs_grid = giphy.renderGrid($("#giphy_grid_in_popover .popover-content")[0]);
+        $("body").on(
+            "keyup",
+            "#giphy-search-query",
+            _.debounce(() => {
+                gifs_grid.remove();
+                gifs_grid = giphy.update_grid_with_search_term();
+            }, 400),
```

Comment:
Can you add a comment documenting the thinking around the debounce here?

Reply:
Okay